### PR TITLE
Disable Tapsell ads for now

### PR DESCRIPTION
**Pull request by atavism**

### CODE REVIEW

**Review by WendelHime — APPROVED** [review]

File: embeddedconfig/global.yaml.tmpl [hunk 1]
```diff
@@ -60,10 +60,10 @@ featuresenabled:
     - label: show-interstitial-ads
       geocountries: af,ax,al,dz,as,ad,ao,ai,aq,ag,ar,am,aw,au,at,az,bs,bh,bd,bb,by,be,bz,bj,bm,bt,bo,bq,ba,bw,bv,br,io,bn,bg,bf,bi,cv,kh,cm,ca,ky,cf,td,cl,cx,cc,co,km,cg,cd,ck,cr,hr,cu,cw,cy,cz,dk,dj,dm,do,ec,eg,sv,gq,er,ee,sz,et,fk,fo,fj,fi,fr,gf,pf,tf,ga,gm,ge,de,gh,gi,gr,gl,gd,gp,gu,gt,gg,gn,gw,gy,ht,hm,va,hn,hu,is,in,id,iq,ie,im,il,it,jm,jp,je,jo,kz,ke,ki,kp,kr,kw,kg,la,lv,lb,ls,lr,ly,li,lt,lu,mo,mg,mw,my,mv,ml,mt,mh,mq,mr,mu,yt,mx,fm,md,mc,mn,me,ms,ma,mz,mm,na,nr,np,nl,nc,nz,ni,ne,ng,nu,nf,mk,mp,no,om,pk,pw,ps,pa,pg,py,pe,ph,pn,pl,pt,pr,qa,re,ro,rw,bl,sh,kn,lc,mf,pm,vc,ws,sm,st,sa,sn,rs,sc,sl,sg,sx,sk,si,sb,so,za,gs,ss,es,lk,sd,sr,sj,se,sy,tw,tj,tz,th,tl,tg,tk,to,tt,tn,tr,tm,tc,tv,ug,ua,ae,gb,us,um,uy,uz,vu,ve,vn,vg,vi,wf,eh,ye,zm,zw,ru
       platforms: android,ios
-  tapsellads:
-  - label: show-tapsell-ads
-    geocountries: ir
-    platforms: android 
+  # tapsellads:
```
Comment (WendelHime) [inline embeddedconfig/global.yaml.tmpl:63]: I guess this was added a couple days/month ago, there's a reason for disabling it?

**Comment by WendelHime** [comment]
Just pushed a fix for the test

File: embeddedconfig/global.yaml.tmpl (hunk 1 above)
Comment (atavism) [reply embeddedconfig/global.yaml.tmpl:63]: Sorry, there's a thread about it on Slack.

**Comment by atavism** [comment]
> Just pushed a fix for the test

Thanks for fixing it, @WendelHime!

File: embeddedconfig/global.yaml.tmpl (hunk 1 above)
Comment (WendelHime) [reply embeddedconfig/global.yaml.tmpl:63]: just saw, thanks for sharing!

**Pull request merged**